leonardo-example.conf

```diff
@@ -110,8 +115,13 @@ auth {
 serviceAccounts {
   # To implement your own service account provisioning, subclass org.broadinstitute.dsde.workbench.leonardo.model.ServiceAccountProvider
   # and specify your concrete class here
-  providerClass = ""
-  config { # Do not remove config, even if empty
```

Comment:
I had previously renamed this key to `providerConfig` to be consistent with the other provider, but forgot to update the example conf.

Reply:
thank you for this change, it's been quietly making me go mad